### ISSUE TITLE
Land Usage: Industrial Infrastructure individual object level cover map

### ISSUE THREAD

**Issue by EgorBEremeev** [serbiancaseforspace/planet-balkan-hackathon#6]
This idea is about building specific purpose land usage classification model and search application.

Particularly, the usage type could be _industrial stocks objects_ or _landfills areas_. So, it is examples of areas of extensive commercial land usage.

The inspiration has come from [here](https://medium.com/google-earth/computer-vision-helps-find-parking-lots-to-save-wildlife-b6ed84cb70c3)

Generally, land usage \ land cover application aimed to support environment protection activities.

But getting an instruments to monitor land usage in such granular level, means on the level of individual industrial object open a way for commercial application such as creating country-wide infrastructure object inventory, logistics optimisation solutions, etc.

So the high-level task to make implementation of this idea are:

- [ ] Build\Get a training dataset of shape-files for industrial stocks\landfills areas

- [ ] Transform shape-files dataset into image-mask dataset

- [ ] Train image segmentation model [one example of implementation ](https://github.com/google/earthengine-api/blob/master/python/examples/ipynb/UNET_regression_demo.ipynb)

- [ ] Apply the trained model on new areas of interests

This is very brief idea description. Welcome to evaluate and contribute!

I'm looking for a team to create\join. About myself.
My personal goal for this hackaton is getting my very first experience with programming geo application and earth observation data processing pipelines. I'm familiar with python, google cloud platform (Functions, Storage, BigQuery), AWS (Storage, Image Rekognition), deep learning (TensorFlow, Keras). I got ARSET introduction training about remote-sensing, NASA Earth observation products, SAR (Sentinel 2) data processing.
